api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly.java

```diff
@@ -75,6 +80,8 @@ public ReportDefinition constructReportDefinition() {
 		reportDefinition.setName(getName());
 		reportDefinition.setDescription(getDescription());
 		reportDefinition.setParameters(getParameters());
+		reportDefinition.setBaseCohortDefinition(genericCohortQueries.getBaseCohort(),
```

Comment:
Wasn't it confirmed with Guimino and Nicholas that adding the baseCohort here doesn't work?

Reply:
Strange, in this case its working. I've checked the numbers. @ningosi?

Reply:
Looks like the SqlCohortDefinition includes the baseCohort when it [evaluates](https://github.com/openmrs/openmrs-module-reporting/blob/331fe70b9a96e5855ccc3bffe18a6fcbfbceb710/api/src/main/java/org/openmrs/module/reporting/cohort/definition/evaluator/SqlCohortDefinitionEvaluator.java#L46-L68), but there might still be a bug.

@ynurmahomed please test by modifying a patient record so that it should be excluded by the baseCohort. We need to be sure about this.

Reply:
@ynurmahomed ideally, by tying a base cohort to a report definition, is supposed to filter all other cohortdefinitions tied to that report by the set of ids coming from the base cohort. For every cohort evaluator, it checks if the base cohort is set and use it to intersect to the respective cohortdefiniton being evaluated, hence adding it at report definition level in our code is supposed to work and give correct values and expected, However, we tested for the TX_Curr using that approach and we happened to get discrepancies in numbers, I am NOT sure if it was only unique to that report or NOT and that is why we included it in a cohort composition and seemed to work.

Reply:
@nthfloor I've removed `SERVICO TARV - CUIDADO` program and `S.TARV: ADULTO INICIAL` encounter type from patient 1138 location 281 (egpaf DB). The baseCohort excludes the patient,  and he does not show up on TX_NEW.